app/web-components/src/server/framework-preset-web-components.ts

```diff
@@ -1,39 +1,5 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
-import { Configuration } from 'webpack';
+import type { Configuration } from 'webpack';
 
 export function webpack(config: Configuration) {
-  config.module.rules.push({
-    test: [
-      new RegExp(`src(.*)\\.js$`),
-      new RegExp(`packages(\\/|\\\\)*(\\/|\\\\)src(\\/|\\\\)(.*)\\.js$`),
-      new RegExp(`node_modules(\\/|\\\\)lit-html(.*)\\.js$`),
-      new RegExp(`node_modules(\\/|\\\\)lit-element(.*)\\.js$`),
-      new RegExp(`node_modules(\\/|\\\\)@open-wc(.*)\\.js$`),
-      new RegExp(`node_modules(\\/|\\\\)@polymer(.*)\\.js$`),
-      new RegExp(`node_modules(\\/|\\\\)@vaadin(.*)\\.js$`),
-    ],
-    use: {
-      loader: require.resolve('babel-loader'),
-      options: {
-        plugins: [
-          require.resolve('@babel/plugin-syntax-dynamic-import'),
-          require.resolve('@babel/plugin-syntax-import-meta'),
-          // webpack does not support import.meta.url yet, so we rewrite them in babel
-          [require.resolve('babel-plugin-bundled-import-meta'), { importStyle: 'baseURI' }],
-        ],
-        presets: [
-          [
-            require.resolve('@babel/preset-env'),
-            {
-              useBuiltIns: 'entry',
-              corejs: 3,
-            },
-          ],
-        ],
-        babelrc: false,
-      },
-    },
-  });
-
```

Comment:
@ndelangen @shilman I would like your feedback about that, I'm not sure if we should still keep it or not

Reply:
I'm not sure either. Perhaps consult the WC guys? Or @brion-fuller ?

Reply:
We will need to keep this for compatibility with webpack 4.  Webpack 4 doesn't support `import.meta`.

Reply:
Ok, I will discard this commit then, thanks @brion-fuller